### DISCUSSION TITLE
[@types/graphql-depth-limit]: `graphql` belongs in devDependencies

### DISCUSSION THREAD

**Discussion by tubbo** [DefinitelyTyped/DefinitelyTyped#58687] [Issues with a @types package]
I might be wrong here, but I don't see any reason for `@types/graphql-depth-limit` to be depending on `graphql`. The type definition has no imports (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/graphql-depth-limit/index.d.ts), but the tests use `graphql`, so maybe we could make this a devDependency? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/graphql-depth-limit/graphql-depth-limit-tests.ts

[a comment by an automated account is not shown]

**Comment by eritikass** [comment]
@tubbo i think you right here, those should be devDependency - happy to accept PR

**Reply by tubbo** [reply]
DefinitelyTyped won't let me add anything to devDependencies :(
